leapp/utils/output.py

```diff
@@ -63,6 +63,8 @@ def report_info(report_paths, log_paths, fail=False):
         sys.stdout.write(pretty_block("END OF REPORT", color=Color.bold if fail else Color.green))
         sys.stdout.write("\n")
 
+    sys.stdout.write("Answerfile has been generated at {}\n".format(answerfile))
```

Comment:
This isn't conditional (no `if answerfile`, like a couple of lines above with `if report_paths`) and it should be, mainly since the answerfile is not generated during an upgrade run.

Also `answerfile_path` as a variable name would be more consistent, although that's just a minor nitpick.